src/types/types.ts

```diff
@@ -116,7 +116,6 @@ export type GuiFiatType = {
 
 export type TransactionListTx = EdgeTransaction & {
```

Comment:
I'm of the opinion that this type belongs in `src/reducers/scenes/TransactionListReducer.ts`. What do you think?